DirectEmployers/MyJobs
# [Pd 2784] loading initial data into table and chart

This is loading the initial starting point data into the table. The data should be for locations which starts with country being displayed. Currently the request is hard coded. Possibly look for that to be dynamic in someway here later.

To Test:

Pull down the repository. You might have to run npm install if you don't have the most up to date version of the package.json
You can view the request, response, and data coming back inside of the Network tab and see that the table data is lined up with Location being displayed as the starting point. Chart is not correct as  there is another task and PR related to that for the starting point as well so disregard the Chart for right now.
